c2corg_ui/templates/i18n.html

```diff
@@ -369,12 +369,25 @@
 <span translate>has created a new outing</span>
 <span translate>has created a new waypoint</span>
 <span translate>has created a new article</span>
+<span translate>has created a new boom</span>
 <span translate>has created a new route</span>
 <span translate>has created a new image</span>
+<span translate>has created a new area</span>
+
 <span translate>has updated the outing</span>
 <span translate>has updated the waypoint</span>
 <span translate>has updated the article</span>
 <span translate>has updated the route</span>
 <span translate>has updated the image</span>
+<span translate>has updated the book</span>
+<span translate>has updated the area</span>
+
+<span translate>has added photos to the route</span>
```

Comment:
photos => images (btw people may upload drawings and schemas, not only photos)

In addition shouldn't you write "has added photos to route" instead? (no "the")